README.md

```diff
@@ -1,24 +1,39 @@
-![conduit][logo]
+![Linkerd2][logo]
 
 [![Build Status][ci-badge]][ci]
 [![GitHub license][license-badge]](LICENSE)
 [![Slack Status][slack-badge]][slack]
 
-:balloon: Welcome to Conduit! :wave:
+:balloon: Welcome to Linkerd2! :wave:
 
-Conduit is an ultralight service mesh for Kubernetes. It features a minimalist
-control plane written in Go, and a native proxy data plane written in
-[Rust][rust] that boasts the performance of C without the heartbleed.
+Note: this project is currently in the middle of a migration from the old name (Conduit) to the new name (Linkerd2). While this transition is ongoing, there will references to Conduit that don't make a lot of sense. Not to worry, we're working on it! For more information, check out the [announcement][announcement].
```

Comment:
Mind wrapping this line at 80 characters to match the wrapping in the rest of this file?

Reply:
Typo here: `there will references` => `there will be references`